src/ngraph/serializer.cpp

```diff
@@ -1120,6 +1120,13 @@ shared_ptr<Node> JSONDeserializer::deserialize_node(json node_js)
             node = make_shared<op::BroadcastLike>(args[0], args[1], initial_axes);
             break;
         }
+        case OP_TYPEID::Bucketize_v3:
+        {
+            element::Type output_type = read_element_type(node_js.at("output_type"));
+            bool with_right_bound = get_or_default<bool>(node_js, "with_right_bound", true);
+            node = make_shared<op::v3::Bucketize>(args[0], args[1], output_type, with_right_bound);
+            break;
+        }
```

Comment:
Don't need this since the op has attribute_visitor.

Reply:
thanks, do i need the serializer tests ?

Reply:
No.